core/src/main/scala/kafka/server/AutoTopicCreationManager.scala

```diff
@@ -101,11 +82,9 @@ class DefaultAutoTopicCreationManager(
   private val inflightTopics = Collections.newSetFromMap(new ConcurrentHashMap[String, java.lang.Boolean]())
 
   override def start(): Unit = {
```

Comment:
It seems like we don't have a strong need for this anymore. We could probably get rid of `shutdown` as well since there's probably not a strong reason to clear the inflight topics if the manager won't be reused anyway.